verifiers/internal/gcb/intoto.go

```diff
@@ -0,0 +1,68 @@
+package gcb
+
+// Copy of github.com/in-toto/in-toto-golang/in_toto/slsa_provenance/v0.1
+// This holds an internal copy of in-toto-golang's structs for
+// SLSA predicates to handle GCB's incompatibility with the
+// published specification. Specifically, GCB provenance currently
+// produces a string for ProvenancePredicate.Recipe.DefinedInMaterial
+// rather than the compliant signed integer.
+
+import "time"
+
+const (
+	// PredicateSLSAProvenance represents a build provenance for an artifact.
+	PredicateSLSAProvenance = "https://slsa.dev/provenance/v0.1"
+)
+
+// ProvenancePredicate is the provenance predicate definition.
+type ProvenancePredicate struct {
+	Builder   ProvenanceBuilder    `json:"builder"`
+	Recipe    ProvenanceRecipe     `json:"recipe"`
+	Metadata  *ProvenanceMetadata  `json:"metadata,omitempty"`
+	Materials []ProvenanceMaterial `json:"materials,omitempty"`
+}
+
+// ProvenanceBuilder idenfifies the entity that executed the build steps.
+type ProvenanceBuilder struct {
+	ID string `json:"id"`
+}
+
+// ProvenanceRecipe describes the actions performed by the builder.
+type ProvenanceRecipe struct {
+	Type string `json:"type"`
+	// DefinedInMaterial can be sent as the null pointer to indicate that
+	// the value is not present.
+	// DefinedInMaterial *int        `json:"definedInMaterial,omitempty"`
```

Comment:
nit: add a short local comment as to why this is commented out. Maybe point to comment at top of file.